curriculum/challenges/chinese/01-responsive-web-design/applied-visual-design/animate-elements-continually-using-an-infinite-animation-count.md

```diff
@@ -11,17 +11,19 @@ dashedName: animate-elements-continually-using-an-infinite-animation-count
 
 之前的关卡里介绍了一些动画属性以及 `@keyframes` 规则的用法。 还有一个常用的动画属性是 `animation-iteration-count`，这个属性允许你控制动画循环的次数。 下面是一个例子：
 
-`animation-iteration-count: 3;`
+```css
+animation-iteration-count: 3;
+```
 
-在这种情况下，动画将在运行 3 次后停止，但可以通过设置该值为 `infinite` 来继续运行动画。
+在这里动画会在运行 3 次后停止，如果想让动画一直运行，可以把值设置成 `infinite`。
 
 # --instructions--
 
-把 `animation-iteration-count` 属性值改成 `infinite`，以使右边的球持续跳跃。
+把 `animation-iteration-count` 属性改成 `infinite`，使右边的球一直跳跃。
 
 # --hints--
 
-`animation-iteration-count` 属性应有一个值 `infinite`。
+`animation-iteration-count` 属性的值应为 `infinite`。
 
 ```js
 assert($('#ball').css('animation-iteration-count') == 'infinite');
```

Comment:
Not sure why this is highlighted red, but it doesn't appear to actually be removed from the file.